src/notifications/components/ActionButton.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import localStyles from './Notification.css'
+
+const ActionButton = props => (
+    <div className={localStyles.button} onClick={props.handleClick}>
```

Comment:
I would expect an `<ActionButton>` component to be built on a `<button>` el rather than `<div>`. Try to think about if there are any other HTML els that would be more semantically appropriate whenever you use a `<div>`. 
And further: to better mimic how a `<button>` works, I would move `props.label` on `props.children`